kinesis/src/main/resources/logback.xml

```diff
@@ -15,7 +15,13 @@
         </layout>
     </appender>
 
+    <appender name="EmitToGraphiteLogbackAppender"
+              class="com.expedia.www.haystack.metrics.appenders.logback.EmitToGraphiteLogbackAppender">
+        <address>${HAYSTACK_GRAPHITE_ADDRESS}</address>
```

Comment:
we are already creating an environment variable for graphite (HAYSTACK_GRAPHITE_HOST) should we reuse this one ?

Reply:
Looks like we're using both...with ADDRESS appearing more often than HOST in our code. I'll have Jason merge this change and we can combine the two if we want. (I prefer ADDRESS, as what's being pointed at is typically not a single host.)